build.js

```diff
@@ -20,23 +16,30 @@ const html = link => `<!DOCTYPE html>
 </html>
 `
 
-const mkdir = async f => {
-  try { await fs.mkdir(f) } catch (e) {}
+async function gracefulmkdir(f) {
+  try { await mkdir(f) } catch (e) {}
+}
+
+async function renderHTML(f) {
+  const fileURL = new URL(f, links)
+  const b = await readFile(fileURL)
+  const d = new URL(`${f}/`, dist)
```

Comment:
This is an interesting thing I want to dig into... basically if you have a URL that does not end with a trailing `/` it will get truncated when calling `new URL()`

since you are using a file name to make a directory the writeDir would work with that path, but subsequent file path creation was making the wrong path. This fixes it but is kinda gross. Want to think through better patterns here. At the very least I personally prefer this as it doesn't require using the path APIs in Node.js at all.